scripts/build-docker-images

```diff
@@ -33,7 +33,8 @@ cleanup(){
 }
 
 mkdir -p ${WORKDIR}/target/release
-cp -v target/${TARGET}/release/cdn-server ${WORKDIR}/target/release
+cp -v target/${TARGET}/release/web-server ${WORKDIR}/target/release
```

Comment:
@nomaxg I suggest merging `main`. I think the new script has a higher chance of working.